.gitignore

```diff
@@ -43,6 +43,7 @@ reports/
 !.env.default
 !package.json
 !package-lock.json
+!tsconfig.json
 node_modules/
```

Comment:
this should be /node_moidules/ once you add yarn i guess

Reply:
node_modules/ is what is ignorig the node_modules inside each workspace. If I change it to /node_modules/ then its not ignoring node_modules.